core/src/components/login/UpdatePassword.vue

```diff
@@ -29,6 +29,9 @@
 					v-model="password"
 					type="password"
 					name="password"
+					autocomplete="current-password"
```

Comment:
```suggestion
					autocomplete="new-password"
```
as it's the place where one sets a new password :grin: